database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -346,13 +356,20 @@ private void executeInitQueries(Connection connection, List<String> initQueries)
    */
   public abstract static class DBSinkConfig extends DBConfig {
     public static final String TABLE_NAME = "tableName";
+    public static final String DB_SCHEMA_NAME = "dbSchemaName";
     public static final String TRANSACTION_ISOLATION_LEVEL = "transactionIsolationLevel";
 
     @Name(TABLE_NAME)
     @Description("Name of the database table to write to.")
     @Macro
     public String tableName;
 
+    @Name(DB_SCHEMA_NAME)
+    @Description("Name of the database schema table to write to.")
+    @Macro
+    @Nullable
+    public String dbSchemaName;
```

Comment:
We should not add this config here in the common base class because even if it is not present in the json it will be displayed as generic property in other plugins using this class.
For eg, even if MySQL plugin does not have this property in the json still it shows here as generic.
![image](https://user-images.githubusercontent.com/88528384/158745441-e02373ed-9e53-4fff-8583-a030ddc81bd5.png)

Instead of this what we can do is add this property only in config class of oracle sink only and have a `getDBSchemaName()` returning `null` as default but will be overriden by oracle sink config class.

Reply:
@seanzhougoogle WDYT?

Reply:
@seanzhougoogle any thoughts here? I certainly don't want these to show up in other plugins. I thought that by restricting them at the widget level they wouldn't show up but they do.

Reply:
done